commons/admin-console/custom-domain-names.md

```diff
@@ -11,80 +11,131 @@ keywords:
-</p>
-<p>
-If you want to redirect your top-level domain name to a subdomain through HTTP (eg ``domain.com`` to ``www.domain.com``), most domain name registrars provide a web redirection service.
-</p>
-</div>
-
+Add it in the application configuration: in the console, click on your **application name** in the first panel, then choose **domain names**. You'll have to choice to add a custom domain name or use a sub-domain under ``*.cleverapps.io``
 
-### Your application runs in the Europe ('PAR') zone
 
-You have two non exclusive choices to add custom personal domain names:
+## Personal Domain Names
 
-  * Sub-domains like ``hello.example.com`` and wildcard sub-domains like ``*.example.com``
 
-    On your domain name provider, point the associated *hello* **CNAME record** to ``domain.par.clever-cloud.com``
+### Your Application Runs in the Europe/Paris ('PAR') Zone
 
-    <a href="http://www.gandi.net" target="_blank">Gandi.net</a> example :
-    <figure class="cc-content-img">
-      <img  src="/doc/assets/images/domain2.png">
-    </figure>
-    <figcaption>CNAME records example</figcaption>
+<table class="table table-bordered" style="text-align:center">
+  <tr>
+    <th><center>Record Type</center></th>
+    <th><center>Value</center></th>
+  </tr>
+  <tr>
+    <td>CNAME <div><span class="label label-success">Recommended</span></div></td>
+    <td>``domain.par.clever-cloud.com``</td>
+  </tr>
+  <tr>
+    <td>A<div><small>Only if CNAME is not available</small></div></td>
+    <td>Two records on ``62.210.112.171`` and ``62.210.92.244``</td>
+  </tr>
+</table>
 
-  * Domains like ``example.com``
+<br/>
 
-    On your domain name provider, point two **A records** to ``62.210.112.171`` and ``62.210.92.244``
+### Your Application Runs in the North-America/Montreal ('MTL') Zone
 
+<table class="table table-bordered" style="text-align:center">
+  <tr>
+    <th><center>Record Type</center></th>
+    <th><center>Value</center></th>
+  </tr>
+  <tr>
+    <td>CNAME <div><span class="label label-success">Recommended</span></div></td>
+    <td>``domain.mtl.clever-cloud.com``</td>
+  </tr>
+  <tr>
+    <td>A<div><small>Only if CNAME is not available</small></div></td>
+    <td>Two records on ``68.71.34.21`` and ``68.71.34.20``</td>
+  </tr>
+</table>
 
-### Your application runs in the North-America ('MTL') zone
 
-You have two non exclusive choices to add custom personal domain names:
+<br/>
 
-  * Sub-domains like ``hello.example.com`` and wildcard sub-domains like ``*.example.com``
+Two kinds of records are supported, Type A and CNAME. **The use of a CNAME record is highly recommended.** The CNAME configuration will ensure that your DNS configuration will be up-to-date whereas A records will require an action for you if the IP adresses have to be changed. We monitor all domain names linked to apps, so obsolete or incorrect configurations will be notified by email by our support team to apps owners.
```

Comment:
You can point your domain name to clever cloud either with a CNAME record or with A records. **The use of a CNAME record is highly recommended**. With a CNAME record, your DNS configuration is always up-to-date. Using A records will require you to keep the DNS configuration up-to-date manually. Domain names linked to clever cloud applications are monitored, so we will send you an email if your DNS configuration is obsolete or incorrect.